config/application.rb

```diff
@@ -12,6 +12,12 @@ class Application < Rails::Application
     # Enable the asset pipeline
     config.assets.enabled = true
 
+<<<<<<< HEAD
```

Comment:
git adds this when there's a merge conflict - you should remove lines that start with `<<<`, `===`, or `>>>`. i'll teach you how to handle more complex merge conflicts